src/styles/v2/variables.scss

```
@@ -34,4 +34,5 @@ $z-index-sing-note: 10;
 $z-index-sing-note-lyric: 20;
 $z-index-sing-pitch: 30;
 $z-index-sing-playhead: 40;
-$z-index-sing-lyric-input: 50;
+$z-index-sing-tool-palette: 50;
+$z-index-sing-lyric-input: 60;
```

あ、別に問題じゃないのですが気づいたので改行入れさせていただきます！
```suggestion
$z-index-sing-lyric-input: 60;
```